lib/internal/process/per_thread.js

```diff
@@ -69,6 +70,7 @@ function refreshHrtimeBuffer() {
   // Use a BigUint64Array in the closure because this is actually a bit
   // faster than simply returning a BigInt from C++ in V8 7.1.
   hrBigintValues = new BigUint64Array(binding.hrtimeBuffer, 0, 1);
+  constrainedMemoryValus = new BigUint64Array(binding.hrtimeBuffer, 0, 1);
```

Comment:
why is this initialized here? IMO this looks unrelated to `hrtime`

Reply:
Thanks, I have updated the code.

Reply:
I think the `constrainedMemory` API can share the memory (`binding.hrtimeBuffer`) ?